doc/docs/master/deploy-manage/manage/s3gateway/deploy-s3gateway-sidecar.md

```diff
@@ -0,0 +1,108 @@
+# Create an S3-enabled Pipeline
+
+If you want to use Pachyderm with such platforms like Kubeflow or
+Apache™ Spark, you need to create an S3-enabled Pachyderm pipeline.
+Such a pipeline ensures that data provenance of the pipelines that
+run in those external systems is properly preserved and is tied to
+corresponding Pachyderm jobs.
+
+Pachyderm can deploy the S3 gateway either together with the `pachd` pod or
+as a sidecar next to your pipeline worker pod. The former is
```

Comment:
This implies that deployment of the master s3g is a flag, but it's not; it's just deployed by default (and last I checked, there isn't a flag to disable it.)

Reply:
Modified.